benchmarks/src/main/scala/zio/IOMapBenchmark.scala

```diff
@@ -113,7 +113,7 @@ class IOMapBenchmark {
       if (n <= 1) t
       else sumTo(t.map(_ + n), n - 1)
 
-    runtime.unsafeRun(sumTo(IO.succeedLazy(0), depth))
+    runtime.unsafeRun(sumTo(IO.succeed(0), depth))
```

Comment:
This should become `effectTotal` in line with other effect types that delay execution in their benchmarks (e.g. monix `Task.eval`)

Reply:
:heavy_check_mark: